### PR TITLE
feat: faqs in frontmatter

### PR DESCRIPTION
**Pull request by PatrickHeneise**

### CODE REVIEW

File: linkerd.io/content/2/faq/_index.md [hunk 1]
```diff
@@ -1,149 +1,68 @@
...
+    answer_schema: Linkerd was originally created by Buoyant.  While Linkerd is a CNCF project, Buoyant continues to be a primary maintainer and sponsor.
+  - question: Who is Linkerd for?
+    answer: Linkerd is for everyone. See [Linkerd's Commitment to Open Governance](https://linkerd.io/2019/10/03/linkerds-commitment-to-open-governance/). In practice, Linkerd has certain technical prerequisites, such as Kubernetes.
+    answer_schema: Linkerd is for everyone. See Linkerd's Commitment to Open Governance. In practice, Linkerd has certain technical prerequisites, such as Kubernetes.
+  - question: How do I pronounce Linkerd?
+    answer: Linkerd rhymes with "Cardi B". The "d" is pronounced separately, as in "Linker-DEE".
+    answer_schema: Linkerd rhymes with "Cardi B". The "d" is pronounced separately, as in "Linker-DEE".
+  - question: How do I write Linkerd?
+    answer: Just like this - Linkerd. Capital "L", lower-case everything else.
+    answer_schema: Just like this - Linkerd. Capital "L", lower-case everything else.
```
Comment (wmorgan) [inline linkerd.io/content/2/faq/_index.md:39]: Can we use a > here to still allow the ":" character? Does that still work?

Comment (PatrickHeneise) [reply linkerd.io/content/2/faq/_index.md:39]: Forestry should add this automatically when needed

**Comment by PatrickHeneise** [comment]
Not sure about the errors from the link check, all of the ids exist

**Review by wmorgan — APPROVED** [review]
lgtm. Thanks. I think there are length limits to the schema answers too, but I can do a wordsmithing pass once this merges.

**Comment by wmorgan** [comment]
@PatrickHeneise can you merge master one last time please?

**Comment by cpretzer** [comment]
The checks for link hashes are failing because there is an incompatibility in the way that `hugo` **anchorizes** titles and what `htmltest` expects when a header is turned into an anchor, especially when there are characters like quotes and slashes

Hugo [generates the anchors](https://gohugo.io/functions/anchorize/) using the [blackfriday](https://github.com/russross/blackfriday#sanitized-anchor-names) library and htmltest has [its own logic](https://github.com/wjdp/htmltest/blob/4d3efc5d8bad48c07b3801e1d6e7730c723e395c/htmldoc/document.go#L111)

**Pull request merged**